### PR TITLE
Fix presentation toolbar not resizing while panel size is changing

### PR DESCRIPTION
**Pull request by KDSBrowne**
This PR will make the presentation toolbar resize while a panel's size is being changed. Currently the toolbar only resizes itself after the user stops changing the panel size.

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/panel-manager/component.jsx [hunk 1]
```diff
@@ -104,8 +104,8 @@ class PanelManager extends Component {
         enable={resizableEnableOptions}
         key={this.userlistKey}
         size={{ width: userlistWidth }}
+        onResize={() => window.dispatchEvent(new Event('resize'))}
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/panel-manager/component.jsx]: You could add to a function and call in each event listener, instead re-write the code.

**Comment by pedrobmarin** [comment]
Hey. A little off-topic here but why is there a `renderWaitingUsersPanelResizable` using the shared notes panel properties?

**Comment by KDSBrowne** [comment]
Not sure, it looks like it was introduced in https://github.com/bigbluebutton/bigbluebutton/pull/6869.

**Comment by pedrobmarin** [comment]
It is possible that this does never break anything. But it would be better to set it as something to be refactored.

**Pull request merged**